src/main/java/rockPaperScissors/player/Player.java

```diff
@@ -0,0 +1,10 @@
+package rockPaperScissors.player;
+
+import rockPaperScissors.move.Move;
+import rockPaperScissors.ui.Ui;
+
+public interface Player {
+
+    Move makeMove(Ui ui);
+
```

Comment:
A bit much whitespace around here 🐱

Reply:
True 😄

Reply:
No, I thought I got it, but I didn't: meaning that this interface is too small?